src/main/java/dk/kb/netarchivesuite/solrwayback/parsers/WarcParser.java

```diff
@@ -267,7 +265,7 @@ else if (headerLine.toLowerCase().startsWith("location:")) {
     }
     else if (headerLine.toLowerCase().startsWith("transfer-encoding:")) {                                      
       String transferEncoding=headerLine.substring(18).trim();
-      log.debug("transfer-encoding:"+transferEncoding);
+      //log.debug("transfer-encoding:"+transferEncoding);
       if (transferEncoding.equalsIgnoreCase("chunked")) {
```

Comment:
This should be a contains as the transfer-encoding can have more than one value: https://developer.mozilla.org/en-US/docs/Web/HTTP/Headers/Transfer-Encoding